src/themes/clean/assets/css/clean.css

```diff
@@ -243,6 +243,10 @@ main a:hover {
     background-color: rgba(55, 55, 55, .8);
 }
 
+.carousel-caption h2 a {
```

Comment:
When I tested this, the css here setting the link to white is overridden by both `_reboot.scss` at line 187 and `clean.css` at line 46.  I have had some trouble building the assets, so that might be part of it, but the `.carousel-caption` class does show up further down the list and the colour set within it is overridden.